shared/reducers/fs.js

```diff
@@ -117,6 +117,14 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
       return state.mergeIn(['flags'], {kbfsInstalling: true})
     case FsGen.localHTTPServerInfo:
       return state.set('localHTTPServerInfo', Constants.makeLocalHTTPServer(action.payload))
+    case FsGen.favoriteIgnore:
+      return state.mergeIn(['pathItems', Types.pathToString(action.payload.path), 'tlfMeta'], {
+        isIgnored: true,
```

Comment:
If we are setting it here, it looks like `favoriteIgnored` isn't necessary anymore?

Reply:
yeah it isn't strictly necessary. I was debating whether to have a pending state, but ended up not using it. WDYT?

Reply:
Oh I mean `favoriteIgnored`, not `favoriteIgnore`. `favoriteIgnored` just doesn't seem to be used anywhere?

Reply:
Yes I know. I was debating whether to display an intermediate state with `favoriteIgnore`, then use `favoriteIgnored` to actually remove it.

Reply:
Oh I see. I guess that could be useful when we start to polish this part. We could have a spinner somewhere. I'm OK with keeping it!

Reply:
Ah too late. Either way is fine!